index.html

```diff
@@ -443,9 +443,11 @@
         WHATWG URL standard [[!URL]].
       </p>
       <p>
-        The term  <dfn><a href=
+        The terms <dfn><a href=
         "https://html.spec.whatwg.org/multipage/webappapis.html#concept-current-everything">
-         current settings object</a></dfn> is defined in [[HTML]].
+        current settings object</a></dfn> and <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/webappapis.html#concept-relevant-everything">relevant
```

Comment:
Should go to https://html.spec.whatwg.org/multipage/webappapis.html#relevant-settings-object